frontend/src/scenes/retention/retentionTableLogic.tsx

```diff
@@ -104,6 +131,10 @@ export const retentionTableLogic = kea<retentionTableLogicType>({
         ],
         actionFilterTargetEntity: [(s) => [s.filters], (filters) => ({ events: [filters.target_entity] })],
         actionFilterReturningEntity: [(s) => [s.filters], (filters) => ({ events: [filters.returning_entity] })],
+        retentionReference: [
+            (selectors) => [selectors.filters],
+            ({ retention_reference = 'total' }) => retention_reference,
```

Comment:
Nitpick: This is a non-obvious way to do the default/handling undefined. Suggestion: `(filters) => filters.retention_reference || 'total'`,